test/spawn-options.spec.js

```diff
@@ -112,6 +116,9 @@ describe('Spawn options', () => {
       })
 
       describe('spawn from a initialized repo', () => {
+        // TODO: wont work on windows until we get `/shutdown` implemented in js-ipfs
+        if (isWindows) { return }
```

Comment:
Please use a `this.skip()`

Reply:
Skip doesn't work inside `describes` only `it` blocks.

Reply:
https://github.com/ipfs/js-ipfsd-ctl/pull/203#discussion_r169578144